sbncode/CAFMaker/CAFMakerParams.h

```diff
@@ -258,6 +258,14 @@ namespace caf
       25.
     };
 
+    // TODO: may need to be thought more about for overlays. For now using this otherwise unused parameter as a hacky way 
+    //       of ignoring some of the truth info for data files (BH).
+    Atom<bool> IsRealData {
+      Name("IsRealData"),
+      Comment("Set true for a data file."),
+      false
```

Comment:
Topic for discussion: do we really want a default value here?
Beside making the transition easier (which may still be a value, but IMO should not make things harder till the end of time), I am not sure there is a good reason to assume the input to be simulation.

Reply:
Happy to change but this would mean we need to add the `false` setting to the current workflows in ICARUS/SBND for CAFs from simulation. I wasn't sure about tampering with the default workflow, but if the sentiment is to go for it then we can 😅